### DISCUSSION TITLE
what could I contribute?

### DISCUSSION THREAD

**Discussion by ashishpoudel523** [KUCC-1997/kucc-mobile#4] [General]
what could I contribute?

**Comment by aadarshadhakalg** [comment]
Hi @ashishpoudel523 , You can contribute in various ways. You can start discussion on new features or existing features of the app. You can file issues and bugs. 

You can also try to solve the existing issue and raise PR. If you are just starting flutter then I would like to suggest you to work on the issue labelled as **good first issue** or **easy fix**.   

The core development team will be helping you in need.
